packages/@okta/vuepress-site/docs/reference/api/zones/index.md

```diff
@@ -293,6 +295,88 @@ curl -X POST \
 }
 ```
 
+### Create a Block List Network Zone
+
+
+<ApiOperation method="post" url="/api/v1/zones" />
+
+Creates a new Block List Network Zone
+
+#### Valid Request Example
```

Comment:
Is there a reason that you are specifying "valid"? Do you plan to provide an invalid request example also? Otherwise, this can just be: Request example